assets/components/prometheus-operator/manifests/Chart.yaml

```diff
@@ -1,21 +1,22 @@
 apiVersion: v1
-description: Provides easy monitoring definitions for Kubernetes services, and deployment and management of Prometheus instances.
-icon: https://raw.githubusercontent.com/prometheus/prometheus.github.io/master/assets/prometheus_logo-cb55bb5c346.png
+appVersion: 0.41.0
+description: Provides easy monitoring definitions for Kubernetes services, and deployment
+  and management of Prometheus instances.
 engine: gotpl
-maintainers:
-  - name: vsliouniaev
-  - name: bismarck
-  - name: gianrubio
-    email: gianrubio@gmail.com
-name: prometheus-operator
-sources:
-  - https://github.com/coreos/kube-prometheus
-  - https://github.com/coreos/prometheus-operator
-  - https://coreos.com/operators/prometheus
-version: 8.3.2
-appVersion: 0.34.0
-tillerVersion: ">=2.12.0"
 home: https://github.com/coreos/prometheus-operator
+icon: https://raw.githubusercontent.com/prometheus/prometheus.github.io/master/assets/prometheus_logo-cb55bb5c346.png
 keywords:
 - operator
 - prometheus
+maintainers:
+- name: vsliouniaev
+- name: bismarck
+- email: gianrubio@gmail.com
+  name: gianrubio
+name: prometheus-operator
+sources:
+- https://github.com/coreos/kube-prometheus
+- https://github.com/coreos/prometheus-operator
+- https://coreos.com/operators/prometheus
+tillerVersion: '>=2.12.0'
+version: 8.15.6
```

Comment:
It seems latest version is 9.3.0 ? https://github.com/helm/charts/blob/master/stable/prometheus-operator/Chart.yaml#L15

Reply:
Oh good catch.

Reply:
I missed to run `helm repo update`.

Reply:
This is done.